src/parallax/fourth/sections/one.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from "./one.module.scss";
+import btnStyles from "../button.module.scss";
+
+const Section1 = () => {
+  return (
+    <section className={styles.one}>
+      <div className={styles.left}>
+        <h1>
+          Air <br />
```

Comment:
Oh the darkside of `<br />`... read this [A11Y study case: Breaking text on a specific word and Screen Readers implications](https://dev.to/a_sandrina_p/a11y-study-case-splitting-text-into-multiple-lines-and-screen-readers-implications-g72)

Reply:
the author rings a bell to me 🤔 TIL, that is a super practical article 🤩